examples/notebooks/make_subset_cube.ipynb

```diff
@@ -0,0 +1,398 @@
+{
```

Comment:
Nice. Nice colours. Maybe you need not print the cube as you have done that before. But you might have good reasons to do that,